deeppavlov/configs/ranking/mltrack_ranker.json

```diff
@@ -0,0 +1,85 @@
+{
+  "dataset_reader": {
+    "class_name": "mltrack_reader",
+    "data_path": "{DOWNLOADS_PATH}/mltrack_data"
+  },
+  "dataset_iterator": {
+    "class_name": "siamese_iterator",
+    "seed": 243
+  },
+  "chainer": {
+    "in": ["x"],
+    "in_y": ["y"],
+    "pipe": [
+      {
+        "class_name": "bert_ranker_preprocessor",
+        "vocab_file": "{DOWNLOADS_PATH}/bert_model/rubert_cased_L-12_H-768_A-12_v1/vocab.txt",
+        "do_lower_case": false,
+        "max_seq_length": 64,
+        "in": ["x"],
+        "out": ["bert_features"]
+      },
+      {
+        "class_name": "bert_ranker",
+        "n_classes": 3,
+        "one_hot_labels": false,
+        "return_probas": false,
+        "bert_config_file": "{DOWNLOADS_PATH}/bert_model/rubert_cased_L-12_H-768_A-12_v1/bert_config.json",
+        "pretrained_bert": "{DOWNLOADS_PATH}/bert_model/rubert_cased_L-12_H-768_A-12_v1/bert_model.ckpt",
+        "save_path": "{MODELS_PATH}/model",
+        "load_path": "{MODELS_PATH}/model",
+        "keep_prob": 0.5,
+        "learning_rate": 2e-05,
+        "learning_rate_drop_patience": 2,
+        "learning_rate_drop_div": 2.0,
+        "in": ["bert_features"],
+        "in_y": ["y"],
+        "out": ["predictions"]
+      }
+    ],
+    "out": ["predictions"]
+  },
+  "train": {
+    "epochs": 3,
+    "batch_size": 36,
+    "metrics": [
+      {
+      "name": "ndcg",
+      "inputs": ["y", "predictions"]
+      }
+    ],
+    "val_every_n_epochs": 3,
+    "evaluation_targets": [],
+    "validate_first": false,
+    "tensorboard_log_dir": "{MODELS_PATH}/"
+  },
+  "metadata": {
+    "variables": {
+      "ROOT_PATH": "~/.deeppavlov",
+      "DOWNLOADS_PATH": "{ROOT_PATH}/downloads",
+      "MODELS_PATH": "{ROOT_PATH}/models/mltrack_model"
```

Comment:
Please use default value for `"MODELS_PATH": "{ROOT_PATH}/models"` to conform to #844  
You can add another variable like `"MODEL_PATH": "{MODELS_PATH}/mltrack_model"`

Reply:
Fixed